### ISSUE TITLE
It's not possible to completly clear the input (back to null) when Auto Decimal Mode is enabled

### ISSUE THREAD

**Issue by MarceloLuis1997** [dm4t2/vue-currency-input#202]
Hello!

I'm using this awesome plugin for a while, and there's something I noticed recently, when `autoDecimalModel` is enabled, it is not possible to completly clear the input (`null`), it keeps the `0,00` (`0`) value.

How to reproduce:

1 - Open the Playground: https://dm4t2.github.io/vue-currency-input/playground/
2 - Enable Auto Decimal Mode
3 - Type any value on the input (i.e. 1234,00) (Number value: 1234)
4 - Keep pressing delete or backspace until it shows 0,00 (Number value: 0)
5 - At this point, the only way to delete the `0,00` value is selecting the text (ctrl + a) and pressing delete or backspace, doing this the value goes back to `null`

Basically, I would like to be able to delete the `0,00` just by deleting the value and assign `null` to the input value again, without having to select all the text. This behavior occurs normally when `autoDecimalMode` is disabled.

**Comment by dm4t2** [comment]
Hi, thanks for reporting. I have to look into it.

**Comment by MarceloLuis1997** [comment]
Hi @dm4t2 ! Any update about this?

**Comment by dm4t2** [comment]
I fixed this in [2.0.0-rc.3](https://github.com/dm4t2/vue-currency-input/releases/tag/2.0.0-rc.3) 🎉 

You can test it on the playground:
https://vue-currency-input-next.netlify.app/playground/

**Comment by MarceloLuis1997** [comment]
It worked! Thank you very much!
